test/go/base64.xsd.go

```diff
@@ -12,14 +12,14 @@ type MyType1 string
 // MyType2 ...
 type MyType2 struct {
 	XMLName    xml.Name `xml:"myType2"`
-	LengthAttr int      `xml:"length,attr,omitempty"`
+	LengthAttr *int     `xml:"length,attr,omitempty"`
```

Comment:
As similar comments in PR [#96](https://github.com/xuri/xgen/pull/96#discussion_r2340300180). In Go language, if we defined field tag with `omitempty` or set field data type as pointer, the element will not in serialization result with zero value or nil, these two indicators is duplicated, we need just choose one of them. Just using pointer.